src/components/ha-expansion-panel.ts

```diff
@@ -132,9 +149,6 @@ class HaExpansionPanel extends LitElement {
 
       .summary-icon {
         transition: transform 150ms cubic-bezier(0.4, 0, 0.2, 1);
-        margin-left: auto;
-        margin-inline-start: auto;
-        margin-inline-end: initial;
```

Comment:
@bramkragten do you remember why you did this?

Reply:
I mean the `margin-inline-start` and `margin-inline-end`

Reply:
That is for Left to Right and Right to Left text

Reply:
@yosilevy Does a lot of these

Reply:
Ok, well I've added a design page now for expansion panel so it's easier to maintain.

Reply:
[Expansion panel gallery](https://deploy-preview-13390--home-assistant-gallery.netlify.app/#components/ha-expansion-panel)